src/foundry/common/documents/active-effect.d.mts

```diff
@@ -282,29 +296,41 @@ declare abstract class BaseActiveEffect<
     options?: LogCompatibilityWarningOptions,
   ): void;
 
-  protected static _onCreateDocuments(
+  /**
+   * @deprecated since v12, will be removed in v14
+   * @remarks "The `Document._onCreateDocuments` static method is deprecated in favor of {@link Document._onCreateOperation | `Document._onCreateOperation`}"
```

Comment:
Nit; you can use `@linkcode`. I suppose you may be avoiding that due to not having the TSDoc change in this PR?

I'll stop re-remarking this now.